api.md

```diff
@@ -287,7 +288,13 @@ Returns error if using an `If-Match` header and the ETag does not match.
 `Accept` - Sets the return type of the updated ACL. Response is parsed `JSON` if `application/json` is explicitly named, otherwise HuJSON will be returned.
 
 ###### POST Body
-ACL JSON or HuJSON (see https://tailscale.com/kb/1018/acls)
+ACL JSON or HuJSON (see https://tailscale.com/kb/1018/acls for more details).
+`Groups` - Static groups of users which can be used for ACL rules.
```

Comment:
The `POST` body should be a JSON or [HuJSON](https://github.com/tailscale/hujson#hujson---human-json) formatted JSON object. An ACL policy may contain the following top-level properties:

* ...
* ...

See https://tailscale.com/kb/1018/acls for more information on those properties.